lesson07/assignment/lesson7/src/main.js

```diff
@@ -0,0 +1,48 @@
+import Vue from 'vue'
+import VueRouter from 'vue-router';
+import App from './App.vue'
+import 'bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './css/style.scss';
+import IndexView from '/Users/stefantmilne/Desktop/uw/html300/html300-v2/lesson07/assignment/lesson7/src/components/IndexView.vue';
```

Comment:
Don't use absolute paths.  This won't work if you go to host your project on a server.  Instead you can use a relative path like: 

import IndexView from './components/IndexView.vue';